qiskit/algorithms/quantum_time_evolution/variational/var_qrte.py

```diff
@@ -0,0 +1,165 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Variational Quantum Real Time Evolution algorithm."""
+
+from typing import Optional, Union, Dict
+
+from scipy.integrate import OdeSolver
+
+from qiskit.algorithms.quantum_time_evolution.evolution_base import EvolutionBase
+from qiskit.algorithms.quantum_time_evolution.variational.error_calculators.gradient_errors.real_error_calculator import (
+    RealErrorCalculator,
+)
+from qiskit.algorithms.quantum_time_evolution.variational.variational_principles.real.real_variational_principle import (
+    RealVariationalPrinciple,
+)
+from qiskit.algorithms.quantum_time_evolution.variational.solvers.ode.abstract_ode_function_generator import (
+    AbstractOdeFunctionGenerator,
+)
+from qiskit.algorithms.quantum_time_evolution.variational.var_qte import VarQte
+from qiskit.circuit import Parameter
+from qiskit.opflow import (
+    OperatorBase,
+    Gradient,
+    StateFn,
+)
+from qiskit.providers import BaseBackend
+from qiskit.utils import QuantumInstance
+
+
+class VarQrte(VarQte, EvolutionBase):
```

Comment:
Maybe I didn't fully understand the inheritance structure here, but shouldn't this be 
```suggestion
class VarQrte(VarQte, Qrte):
```
since it's both real time evolution (= kind of time evolution) and variational (= method of time evolution)? If this inherits from `Qrte`, it also has the advantage that an algorithm can query whether this class does real or imaginary time evo.

Reply:
Correct, thanks. It was overlooked.